### DISCUSSION TITLE
Table => Column Resizable, Column Reorderable and Row Reorderable

### DISCUSSION THREAD

**Discussion by Imadozone** [quasarframework/quasar#15368] [Ideas / Proposals]
Hey guys,

It would be nice to have these features (saw them in VuePrimeface: 
  https://primevue.org/datatable/reorder
  https://primevue.org/datatable/colresize )

On table level:
:ColumnReorder="true"
:RowReorder ="true"

On column level:
:IsResizable="true"

**Many thanks in advance!**

**Comment by zhongjiaguo001** [comment]
You can check out this example https://codesandbox.io/p/sandbox/confident-mccarthy-rcnc4d?file=%2Fsrc%2Fboot%2Fdirectives%2Fresizable.js%3A4%2C24
